app/models/policies/further_education_payments/eligible_fe_provider.rb

```diff
@@ -1,13 +1,57 @@
 module Policies
   module FurtherEducationPayments
     class EligibleFeProvider < ApplicationRecord
+      module ClaimScopes
+        def unverified
+          merge(
+            Policies::FurtherEducationPayments::Eligibility
+              .where(provider_verification_completed_at: nil)
+              .where(repeat_applicant_check_passed: true)
+          )
+        end
+
+        def verified
+          merge(
+            Policies::FurtherEducationPayments::Eligibility
+              .where.not(provider_verification_completed_at: nil)
+          )
+        end
+
+        def verification_overdue
+          where(claims: {created_at: ..2.weeks.ago})
```

Comment:
not 100% sure but does this need to be a proc thing? otherwise in 2 weeks time this will be `Time.now`? I don't know if these things are scopes or dynamically evaluated every time